salt/__init__.py

```diff
@@ -3,9 +3,13 @@
 """
 
 import importlib
+import logging
+import os
```

Comment:
```suggestion
import os
```

Reply:
This is still valid, remove logging.

Reply:
Missed left over from debugging, it is gone